.github/workflows/test_suite_ubuntu.yml

```
@@ -0,0 +1,72 @@
+# Workflow to run CAM-ML test suite
+name: TestSuiteUbuntu
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on pushes to the "main" branch, i.e., PR merges
+  push:
+    branches: [ "main" ]
+
+  # Triggers the workflow on pushes to open pull requests with code changes
+  pull_request:
+    paths:
+      - '.github/workflows/test_suite_ubuntu.yml'
+      - '**.c'
+      - '**.cpp'
+      - '**.fypp'
+      - '**.f90'
+      - '**.F90'
+      - '**.pf'
+      - '**.py'
+      - '**.sh'
+      - '**CMakeLists.txt'
+      - '**requirements.txt'
+      - '**data/*'
```

We do not have a number of these filetypes, or do not use them for the Fortran tests so can remove.
```suggestion
      - '.github/workflows/test_suite_ubuntu.yml'
      - '**.f90'
      - '**.F90'
      - '**CMakeLists.txt'
      - '**requirements.txt'
```